apps/docs/src/components/Icons/IconModal.tsx

```diff
@@ -0,0 +1,89 @@
+import { h } from 'preact';
+import type { StateUpdater } from 'preact/hooks';
+import render from 'preact-render-to-string';
+import classNames from 'classnames';
+
+import { parseToReact } from '@selleo/core/src/parser';
```

Comment:
`render` and `parseToReact` break the build since they're server functions and should be done in advance in buildtime or serverside. So in other words you have to have the strings prepared with components before the render by using doing it in astro buildtime

Reply:
@DawidHaratyk

Reply:
Yes, it's already done :)